Mugs_index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Filling Cups with Liquids</title>
+    <link href="style.css" rel="stylesheet" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  </head>
+  <body>
+    <h1 class="h1">Filling Cups with Liquids</h1>
+
+    <div class="cup">
+      <div class="liquid" id="liquid1"></div>
+    </div>
+    <div>
+      <select id="select1">
+        <option value="coffee">Coffee</option>
+        <option value="tea">Tea</option>
+        <option value="juice">Juice</option>
+      </select>
+    </div>
+    <div><button class="button" onclick="fillCup(1)">Fill Cup</button></div>
+    <div><button class="button" onclick="emptyCup(1)">Empty Cup</button></div>
+
+    <div class="cup">
+      <div class="liquid" id="liquid2"></div>
+    </div>
+    <div>
+      <select id="select2">
+        <option value="coffee">Coffee</option>
+        <option value="tea">Tea</option>
+        <option value="juice">Juice</option>
+      </select>
+    </div>
+    <div><button class="button" onclick="fillCup(2)">Fill Cup</button></div>
+    <div><button class="button" onclick="emptyCup(2)">Empty Cup</button></div>
+
+    <div class="cup">
+      <div class="liquid" id="liquid3"></div>
+    </div>
+
+    <div>
+      <select id="select3">
+        <option value="coffee">Coffee</option>
+        <option value="tea">Tea</option>
+        <option value="juice">Juice</option>
+      </select>
+    </div>
+    <div><button class="button" onclick="fillCup(3)">Fill Cup</button></div>
+    <div><button class="button" onclick="emptyCup(3)">Empty Cup</button></div>
+
+    <script>
+      let liquidHeight1 = 0;

```

Comment:
Instead of this procedural style data - we should use our Cup model. 
Please encapsulate this data and provide a constructor to create such an object - for now please keep it simple as much as you can